src/smart-components/order-process/order-processes.js

```diff
@@ -0,0 +1,190 @@
+import TopToolbar from '../../presentational-components/shared/top-toolbar';
+import { defaultSettings } from '../../helpers/shared/pagination';
+import asyncFormValidator from '../../utilities/async-form-validator';
+import { scrollToTop } from '../../helpers/shared/helpers';
+import TableEmptyState from '../../presentational-components/shared/table-empty-state';
+import { useIntl } from 'react-intl';
+import orderProcessesMessages from '../../messages/order-processes.messages';
+import tableToolbarMessages from '../../messages/table-toolbar.messages';
+
+const columns = (intl) => [
+  {
+    title: intl.formatMessage(tableToolbarMessages.name),
+    transforms: [sortable]
+  },
+  {
+    title: intl.formatMessage(orderProcessesMessages.description),
+    transforms: [sortable]
+  },
+  {
+    title: intl.formatMessage(orderProcessesMessages.created_at),
+    transforms: [sortable]
+  }
+];
+
+const debouncedFilter = asyncFormValidator(
+  (filter, dispatch, filteringCallback, meta = defaultSettings) => {
+    filteringCallback(true);
+    dispatch(setFilterValueOrderProcesses(filter, meta));
+    return dispatch(fetchOrderProcesses(meta)).then(() =>
+      filteringCallback(false)
+    );
+  },
+  1000
+);
+
+const prepareChips = (filterValue, intl) =>
+  filterValue
+    ? [
+        {
+          category: intl.formatMessage(tableToolbarMessages.name),
+          key: 'name',
+          chips: [{ name: filterValue, value: filterValue }]
+        }
+      ]
+    : [];
+
+const initialState = (filterValue = '') => ({
+  filterValue,
+  isOpen: false,
+  isFetching: true,
+  isFiltering: false
+});
+
+const orderProcessesState = (state, action) => {
+  switch (action.type) {
+    case 'setFetching':
+      return { ...state, isFetching: action.payload };
+    case 'setFilterValue':
+      return { ...state, filterValue: action.payload };
+    case 'setFilteringFlag':
+      return { ...state, isFiltering: action.payload };
+    default:
+      return state;
+  }
+};
+
+const OrderProcesses = () => {
+  const {
+    orderProcesses: { data, meta },
+    sortBy,
+    filterValueRedux
+  } = useSelector(
+    ({
+      orderProcessReducer: {
+        orderProcesses,
+        sortBy,
+        filterValue: filterValueRedux
+      }
+    }) => ({ orderProcesses, sortBy, filterValueRedux }),
+    shallowEqual
+  );
+  const [{ filterValue, isFetching, isFiltering }, stateDispatch] = useReducer(
+    orderProcessesState,
+    initialState(filterValueRedux)
+  );
+
+  const dispatch = useDispatch();
+  const intl = useIntl();
+
+  const updateOrderProcesses = (pagination) => {
+    stateDispatch({ type: 'setFetching', payload: true });
+    return dispatch(fetchOrderProcesses(pagination))
+      .then(() => stateDispatch({ type: 'setFetching', payload: false }))
+      .catch(() => stateDispatch({ type: 'setFetching', payload: false }));
+  };
+
+  useEffect(() => {
```

Comment:
you should also highlight the chrome navigation when landing in this page directly.
`insights.chrome.appNavClick({ id: 'page-id', secondaryNav: true });`